source/package-json.d.ts

```diff
@@ -211,6 +211,29 @@ declare namespace PackageJson {
 		[packageName: string]: string;
 	}
 
+	/**
+	Conditions which provide a way to resolve a package entry point based on the environment.
+	*/
+	export type ExportCondition = LiteralUnion<
+		| 'import'
+		| 'require'
+		| 'node'
+		| 'deno'
+		| 'browser'
+		| 'electron'
+		| 'react-native'
```

Comment:
@kripod

Can you reference where the values `browser`, `electron`, `react-native` are from? AFAIK they're unofficial and these values depend on specific bundlers (eg. Webpack)

Reply:
Thank you for asking. Unfortunately, I’m not sure anymore. It could possibly be from (an older version of) the Node.js docs.